freqtrade/commands/analyze_commands.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import os
+from pathlib import Path
+from typing import Any, Dict
+
+from freqtrade.configuration import setup_utils_configuration
+from freqtrade.enums import RunMode
+from freqtrade.exceptions import OperationalException
+
+
+logger = logging.getLogger(__name__)
+
+
+def setup_analyze_configuration(args: Dict[str, Any], method: RunMode) -> Dict[str, Any]:
+    """
+    Prepare the configuration for the entry/exit reason analysis module
+    :param args: Cli args from Arguments()
+    :param method: Bot running mode
+    :return: Configuration
+    """
+    config = setup_utils_configuration(args, method)
+
+    no_unlimited_runmodes = {
+        RunMode.BACKTEST: 'backtesting',
+    }
+    if method in no_unlimited_runmodes.keys():
+        from freqtrade.data.btanalysis import get_latest_backtest_filename
+
+        btfile = get_latest_backtest_filename(config['user_data_dir'] / 'backtest_results')
+        signals_file = f"{os.path.basename(os.path.splitext(btfile)[0])}_signals.pkl"
```

Comment:
Please use `pathlib.Path` instead of `os.path` - it's ... more resiliant for multi-os scenarios (windows in particular)